x/dex/keeper/cancel_limit_order.go

```diff
@@ -30,23 +30,25 @@ func (k Keeper) CancelLimitOrderCore(
 		coinsOut,
 	)
 	if err != nil {
-		return err
+		return sdk.Coin{}, sdk.Coin{}, err
 	}
 
-	// This will never panic since TradePairID has already been successfully constructed by ExecuteCancelLimitOrder
-	pairID := tradePairID.MustPairID()
+	makerDenom := makerCoinOut.Denom
+	takerDenom := takerCoinOut.Denom
+	// This will never panic since TradePairID has already been successfully constructed by ExecuteWithdrawFilledLimitOrder
+	pairID := types.MustNewPairID(makerDenom, takerDenom)
```

Comment:
Outdated comment, there is no TradePairID anymore. And as we disscused, need to fix pair creation.